marimo/_output/formatters/tqdm_formatters.py

```diff
@@ -35,7 +35,9 @@ def tqdm_to_progress_bar(
                     total = args[2]
 
                 return progress_bar(
-                    collection=iterable, title=desc, total=total
+                    collection=iterable,
+                    title=desc or "Loading...",
```

Comment:
Since this is optional, it's somewhat ugly when nothing is passed and taking a while to load, it looks like marimo is stuck (noticed in the hf example)